### ISSUE TITLE
false positive on `classed` helper from tw-classed with template literals

### ISSUE THREAD

**Issue by tleunen** [sergioazoc/oxlint-tailwindcss#9]
The following is triggering an error about unknown classes with "div"
```ts
export const Component = classed(`div`, 'truncate')
```
```
"div" is not a valid Tailwind class. Did you mean "grid"?oxc(tailwindcss(no-unknown-classes))
```

When using `div` in single quotes though, everything is fine.
```ts
export const Component = classed('div', 'truncate')
```

**Comment by sergioazoc** [comment]
Fixed in v0.4.1.

The `extractFromClassedCall()` function only checked for `Literal` and `Identifier` nodes when skipping the first argument (element type). A template literal (backticks) generates a `TemplateLiteral` AST node which wasn't recognized, so it fell through to the class extractor and treated `"div"` as a Tailwind class.

The fix adds a `TemplateLiteral` check to the skip logic. All three forms now work correctly:

```ts
classed("div", "truncate")   // ✅ string literal
classed(`div`, "truncate")   // ✅ template literal (was broken)
classed(Button, "truncate")  // ✅ component reference
```

Thanks for the report!
